src/plasma/plasma_manager.cc

```diff
@@ -528,6 +536,11 @@ void PlasmaManagerState_free(PlasmaManagerState *state) {
   delete state;
 }
 
+bool is_object_received(PlasmaManagerState *state, ObjectID object_id){
```

Comment:
`const ObjectID &object_id`

Reply:
If you const the 2nd arg, it leaves a hint that the 1st arg is an out-arg, which doesn't seem to be the case.  So please const both :)